### PR TITLE
[FEA] resource add REP and update roscon.

### PR DESCRIPTION
**Pull request by ZhenshengLee**
@carlossvg Could you give this a CR?

### CODE REVIEW

**Review by ZhenshengLee — COMMENTED** [review]
LGTM

**Comment by carlossvg** [comment]
@ZhenshengLee Thank you for the PR!

**Review by carlossvg — APPROVED** [review]

File: source/Resources/resources.md [hunk 1]
```diff
@@ -15,12 +15,16 @@ This document contains a compilation of ROS and real-time related documents, art
...
+
+- [REP-2014: Benchmarking performance in ROS 2](https://ros.org/reps/rep-2014.html)
+- [DRAFT-REP-2017: Thread attributes configuration support in rcl](https://github.com/ros-infrastructure/rep/pull/385)
 
 ## ROSCon
 
 ### ROSCon 2015
 
-- Real-time Performance in ROS 2 [Slides](https://roscon.ros.org/2015/presentations/RealtimeROS2.pdf) [Video](https://vimeo.com/142621778)
+- Real-time control in ROS and ROS 2.0 [Slides](https://roscon.ros.org/2015/presentations/RealtimeROS2.pdf) [Video](https://vimeo.com/142621778)
```
Comment (christophebedard) [inline source/Resources/resources.md:27]: ROS 2.0 -> ROS 2

Comment (christophebedard) [reply source/Resources/resources.md:27]: I see that this is simply the presentation's title (dating back from 2015), so I guess this is fine

Comment (ZhenshengLee) [reply source/Resources/resources.md:27]: OK

**Pull request merged**